tests/Integration/Behaviour/Features/Context/Domain/TaxFeatureContext.php

```diff
@@ -277,9 +275,36 @@ public function addTaxRuleGroupToTax(string $taxGroupReference, string $taxRefer
         $taxRule = new TaxRule();
         $taxRule->id_tax = $tax->id;
         $taxRule->id_tax_rules_group = $taxRulesGroup->id;
-        $taxRule->behavior = 1;
+        $taxRule->behavior = 0;
```

Comment:
This changes the initial behaviour, although I'm not sure if the value 0 is correctly handled when I look into the code of TaxCalculator, seems like it can only handle `TaxCalculator::COMBINE_METHOD` (1) or `TaxCalculator::ONE_AFTER_ANOTHER_METHOD` (2) but doesn't really handle the 0 value which is "this tax only"

Reply:
I looked into it and all the test where using one tax only, so it didn't matter what behavior is used, but i think by default it makes more sense to test the default logic that is 0- one tax only and for 1-tax combined use specific tests.